src/pages/JazzBar/JazzBarSkeletonPage/JazzBarSkeletonPage.tsx

```diff
@@ -69,14 +78,74 @@ const JazzBarSkeletonPage: React.FunctionComponent<PropsType> = ({
             {userList && (
               <div className="user-list">
                 {selectedTab === "jazz" ? (
-                  <TablesUserList
-                    experienceName="kansassmittys"
-                    limit={24}
-                    setUserList={setUserList}
-                  />
+                  <>
+                    <div className="row header no-margin">
+                      <p>
+                        <span className="bold">
+                          {users &&
+                            users.filter(
+                              (user: User) =>
+                                user.data &&
+                                user.data["kansassmittys"] &&
```

Comment:
All the `kansassmittys` have to become `Jazz Mountain` for the attendance count to work